src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -150,9 +151,9 @@ class IncidentsListContainer extends React.Component<Props> {
     const {pathname, query} = location;
     const {orgId} = params;
 
-    const openIncidentsQuery = {...query, status: 'open'};
-    const closedIncidentsQuery = {...query, status: 'closed'};
-    const allIncidentsQuery = {...query, status: 'all'};
+    const openIncidentsQuery = omit({...query, status: 'open'}, 'cursor');
```

Comment:
We could do something like this to omit `cursor` once, instead of for each query:

```javascript
const {cursor: _, ...query} = location.query || {};
```

This destructures `cursor` from `location.query`, and we get the rest of `location.query` in `query` local var.